### PR TITLE
doc(FAQ): guidance to layout 3 column content on 2x grid

### PR DESCRIPTION
**Pull request by shixiedesign**
Many questions have came in on How to layout 3 column content on our new 16 column grid. This PR adds a section to the FAQ page with image & video to explain the options. 

**Review link** 
https://carbon-website-git-shixie-faq-3col-layout-guide.carbon-design-system.now.sh/help/faq#how-do-i-migrate-an-existing-design-from-a-12-column-grid-to-the-16-column-grid

### CODE REVIEW

File: src/pages/help/faq/index.mdx [hunk 1]
```diff
@@ -93,3 +93,15 @@ Carbon v9.x
 Carbon charts are in development and classified as experimental. Check out the [experimental charts repo](https://carbon-design-system.github.io/carbon-charts/?experimental=true) to explore the work so far.
 
 These charts will receive an updated color palette to match Carbon v10. Visual guidance and chart components are also in the works!
+
+### How do I migrate an existing design from a 12 column grid to the new 16 column grid?
+
+There are many acceptable ways to move content from a 12 column grid onto a 16 column grid. If keeping the number of columns unchanged is not important to you, see below examples.
```
Comment (connor-leech) [inline src/pages/help/faq/index.mdx]:
```suggestion
There are many ways to move content from a 12 column grid onto a 16 column grid. If keeping the number of columns unchanged is not important to you, see below examples.
```

File: src/pages/help/faq/index.mdx [hunk 2]
```diff
@@ -93,3 +93,15 @@ Carbon v9.x
...
+
+<Video vimeoId="356743483" />
+<Video vimeoId="356743492" />
+<Video vimeoId="356743508" />
+
+If keeping the number of columns to 3 is somehow significant to you, here are some other options.
```
Comment (connor-leech) [inline src/pages/help/faq/index.mdx]:
```suggestion
If keeping the number of columns to 3 is essential, here are some other options.
```

File: src/pages/help/faq/index.mdx [hunk 3]
```diff
@@ -93,3 +93,15 @@ Carbon v9.x
 Carbon charts are in development and classified as experimental. Check out the [experimental charts repo](https://carbon-design-system.github.io/carbon-charts/?experimental=true) to explore the work so far.
 
 These charts will receive an updated color palette to match Carbon v10. Visual guidance and chart components are also in the works!
+
+### How do I migrate an existing design from a 12 column grid to the new 16 column grid?
```
Comment (connor-leech) [inline src/pages/help/faq/index.mdx]:
```suggestion
### How do I migrate an existing design from a 12 column grid to the 16 column grid?
```

File: src/pages/help/faq/index.mdx (hunk 2 above)
Comment (shixiedesign) [reply src/pages/help/faq/index.mdx]: Hahahah was I too snarky?

**Pull request merged**